src/entities/Seed.ts

```diff
@@ -403,6 +409,69 @@ export class Seed {
       });
   }
 
+  async addWhitelist(whitelistAddress: Set<Address>): Promise<TransactionReceipt> {
+    return this.transactionsService.send(
+      () => this.contract.whitelistBatch([...whitelistAddress])
+    ).then((receipt) => {
+      if(receipt){
+        this.hydrateUser()
+        return receipt
```

Comment:
please make sure you run lint on your files.

Reply:
👍

Reply:
done